app/views/provider_interface/account_creation_in_progress.html.erb

```diff
@@ -6,13 +6,24 @@
       Your account is not ready yet
```

Comment:
Potentially this line could end up being confusing given the proposed changes

Reply:
I agree. Happy to change this but let's wait for the final version of the copy first.